lib/screens/v2/widget_instance/cr_departures.ex

```diff
@@ -0,0 +1,150 @@
+defmodule Screens.V2.WidgetInstance.CRDepartures do
+  @moduledoc false
+
+  alias Screens.Config.V2.CRDepartures
+  alias Screens.V2.Departure
+  alias Screens.V2.WidgetInstance.CRDepartures, as: CRDeparturesWidget
+
+  defstruct config: nil,
+            departures_data: [],
+            destination: nil
+
+  @type t :: %__MODULE__{
+          config: CRDepartures.t(),
+          departures_data: list(Departure.t()),
+          destination: String.t()
+        }
+
+  defimpl Screens.V2.WidgetInstance do
+    def priority(%CRDeparturesWidget{config: config}), do: config.priority
+
+    def serialize(%CRDeparturesWidget{
+          config: config,
+          departures_data: departures_data,
+          destination: destination
+        }) do
+      %{
+        departures:
+          Enum.map(
+            departures_data,
+            &CRDeparturesWidget.serialize_departure(&1, config.wayfinding_arrows)
+          ),
+        destination: destination,
+        time_to_destination: config.travel_time_to_destination
+      }
+    end
+
+    def slot_names(_instance), do: [:main_content_left]
+
+    def widget_type(_instance), do: :cr_departures
+
+    def valid_candidate?(_instance), do: true
+
+    def audio_serialize(%CRDeparturesWidget{
+          departures_data: departures_data,
+          config: config,
+          destination: destination
+        }) do
+      %{
+        departures:
+          Enum.map(
+            departures_data,
+            &CRDeparturesWidget.serialize_departure(&1, config.wayfinding_arrows)
+          ),
+        destination: destination,
+        time_to_destination: config.travel_time_to_destination
+      }
+    end
+
+    def audio_sort_key(_instance), do: [1]
+
+    def audio_valid_candidate?(_instance), do: true
+
+    def audio_view(_instance), do: ScreensWeb.V2.Audio.CRDeparturesView
+  end
+
+  def serialize_departure(%Departure{} = departure, wayfinding_arrows, now \\ DateTime.utc_now()) do
```

Comment:
Normally when a widget requires a `now` value, we add it as a field on the struct ([example](https://github.com/mbta/screens/blob/master/lib/screens/v2/widget_instance/alert.ex#L19)) so that all logic inside the widget calculates time-based stuff from the same `now`. If there's time, please do that, or make a tech-debt ticket if not.

Reply:
Ah, this issue is also in Departures